source/chapter2-source-file-format.rst

```diff
@@ -280,6 +280,7 @@ os
     arm-trusted-firmware  ARM Trusted Firmware
     dell                  Dell
     efi                   EFI Firmware
+    elf                   ELF Image
```

Comment:
This seems like more of a type than an OS?

Reply:
In this case, it is still an OS, since it should run and be processed as an OS.
Also in the U-Boot project code it is an OS.

Reply:
Images like efi, arm-trusted-firmware, dell and others also isn't OS

Reply:
In my case, it's exactly the OS (like seL4 or other non-opensource OS)

Reply:
OK, so how about adding your OS as another entry? You can still use elf, but I believe that is better as the subtype